QoL/Modules/SpeedBroke.cs

```diff
@@ -58,6 +67,7 @@ public override void Initialize()
             On.InputHandler.Update += EnableSuperslides;
             ModHooks.ObjectPoolSpawnHook += OnObjectPoolSpawn;
             USceneManager.activeSceneChanged += SceneChanged;
+            _elevatorStorageHook = new(_DieFromHazardIteratorMethod, RestoreElevatorStorage);
```

Comment:
fwiw I'd only use the target-typed new when the type is obvious so I'd just have new ILHook here given that it's away from the field declaration